ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,10 @@ Having problems? Have you checked these first:
 * If not, please update then uninstall the TinyTDS gem and re-install it.
 * Using Ubuntu? If so, you may have forgotten to install FreeTDS first.
 * Doing work with threads and the raw client? Use the ConnectionPool gem?
+* On Windows? If so, do you need devkit for your ruby install?
```

Comment:
Can we move this to the first line? Idea being put these in order we think are most issues first.

Can we change the first line from `Having problems? Have you checked these first:` to something like this with BOLD markdown?

**PLEASE CHECK THESE FIRST**